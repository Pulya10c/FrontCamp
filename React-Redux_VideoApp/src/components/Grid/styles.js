import styled from 'styled-components';
import { mediaMd, mediaSm } from 'utils/style-utils';

export const Grid = styled.section`
  display: grid;
  grid-template-columns: repeat(5, 1fr);
  grid-gap: 5px;
  font-size: 20px;
  padding: 5px;

  ${mediaMd`
    grid-template-columns: repeat(3, 1fr);
    font-size: 24px;
  `};

  ${mediaSm`
    grid-template-columns: repeat(1, 1fr);
    font-size: 20px;
  `};
`;
