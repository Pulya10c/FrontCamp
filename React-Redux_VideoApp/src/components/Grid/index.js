import { Grid } from './styles';

export default Grid;
