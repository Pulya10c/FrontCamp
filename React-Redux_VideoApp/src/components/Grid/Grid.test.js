import React from 'react';
import { shallow } from 'enzyme';
import Grid from './';

describe('Component Grid', () => {
  it('should render correctly', () => {
    expect(shallow(<Grid />)).toMatchSnapshot();
  });
});
