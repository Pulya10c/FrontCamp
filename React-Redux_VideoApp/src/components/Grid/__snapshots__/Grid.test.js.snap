// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Grid should render correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  display: grid;
  grid-template-columns: repeat(5, 1fr);
  grid-gap: 5px;
  font-size: 20px;
  padding: 5px;

  ",
          "
  @media (max-width: ",
          "768",
          "px) {
    ",
          "
    grid-template-columns: repeat(3, 1fr);
    font-size: 24px;
  ",
          ";
  }
",
          ";

  ",
          "
  @media (max-width: ",
          "576",
          "px) {
    ",
          "
    grid-template-columns: repeat(1, 1fr);
    font-size: 20px;
  ",
          ";
  }
",
          ";
",
        ],
      },
      "displayName": "styled.section",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "section",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
